examples/lifelong_learning/robot_dog_delivery/README.md

```diff
@@ -1,201 +1,226 @@
-# Using Lifelong Learning in Campus Robot Delivery Scenario
-
-## Introduction
-This example introduces how to use Sedna lifelong learning to implement the lifelong learning delivery task of the robot in the campus.
-
-This demo mainly shows:
-1. Lifelong learning unseen task recognition algorithm (prediction) can identify and collect unseen task samples.
-2. Lifelong learning unseen task recognition algorithm (detection) can trigger alarms to remind robot admin that emergency response is required.
-3. Lifelong learning unseen task training algorithm improves the inference accuracy of known categories and the ability to identify new categories.
-
-## System Architecture
-
-### Install Sedna
-Follow the [Sedna installation document](/docs/setup/install.md) to install Sedna.
-
-### Prepare Dataset 
-Step1: download [data.txt](https://kubeedge.obs.cn-north-1.myhuaweicloud.com:443/sedna-robo/data.txt?AWSAccessKeyId=EMPTKHQUGPO2CDUFD2YR&Expires=1697698966&Signature=s42sKZVewIP/kgLc4dEzjNCRXfk%3D)
-and upload it to a specfied s3 directory. In this example, the directory is s3://kubeedge/sedna-robo/sedna_data.
-
-Step2: download and decompress [sedna_data.zip](https://kubeedge.obs.cn-north-1.myhuaweicloud.com:443/sedna-robo/sedna_data.zip?AWSAccessKeyId=EMPTKHQUGPO2CDUFD2YR&Expires=1697700463&Signature=9OlCC8qBcQr8LfX1ptbsr7nhU5s%3D), 
-then upload the training images in it to the s3 directory where data.txt is stored.
-
-### Prepare Images
-This example uses these images:
-
-1. training worker:  swr.cn-south-1.myhuaweicloud.com/sedna/sedna-robo:v0.1.1
-2. evaluate worker:  swr.cn-south-1.myhuaweicloud.com/sedna/sedna-robo:v0.1.1
-3. inference worker: swr.cn-south-1.myhuaweicloud.com/sedna/sedna-robo-infer:v0.1.1
-
-These images are generated by the script [build_images.sh](/examples/build_image.sh).
-
-### Create Lifelong Learning Job
-
-#### Preparations
-Before this, user must provide an s3 directory for storage which is denoted as "s3_prefix" in this example, by setting environment variable s3_prefix.
-```
-s3_prefix=$s3_prefix
-cloud_image=swr.cn-south-1.myhuaweicloud.com/sedna/sedna-robo:v0.1.1

```

Comment:
Add an introduction to the images (as introduced below

Reply:
done